extensions-contrib/opentelemetry-extensions/src/main/java/org/apache/druid/data/input/opentelemetry/protobuf/OpenTelemetryMetricsProtobufReader.java

```diff
@@ -153,7 +153,7 @@ private List<InputRow> parseMetric(Metric metric, Map<String, Object> resourceAt
       case HISTOGRAM:
       case SUMMARY:
       default:
-        log.warn("Metric type " + metric.getDataCase() + " is not supported or deprecated.");
+        log.trace("Metric type " + metric.getDataCase() + " is not supported or deprecated.");
```

Comment:
since this is could be called on many rows of data, we should not incur the cost of concatenating strings with every message. To avoid this you can use the built-in placeholders, e.g.

```
log.trace("Metric type {} is not supported", metric.getDataCase())
```